handbook/engineering/extensibility/index.md

```diff
@@ -0,0 +1,66 @@
+# Extensibility team
+
+The extensibility team owns our code host and third-party integrations (including our browser extension) and our [Sourcegraph extensions](https://docs.sourcegraph.com/extensions).
+
+## Contact
+
+- [#extensibility-chat](https://app.slack.com/client/T02FSM7DL/C01LZKLRF0C) channel or @extensibility on Slack
+- On GitHub, mention or assign issues to the [@sourcegraph/extensibility](https://github.com/orgs/sourcegraph/teams/extensibility) team.
+
+## Ownership
+
+This team is responsible for:
+
+1. Bringing the value of Sourcegraph to other developer tools
+1. Bringing the value of other developer tools into Sourcegraph
+
+The team's areas of responsibility include:
+
+- Sourcegraph extensions
+  - Providing a Sourcegraph extension API that enables developers to bring data from their favorite developer tools into their Sourcegraph workflow.
+  - Building useful Sourcegraph extensions on top of the Sourcegraph extension API (for example: Codecov, Datadog, Sentry, Lightstep).
```

Comment:
```suggestion
  - Building and maintaining useful Sourcegraph extensions on top of the Sourcegraph extension API ([Sourcegraph maintained extensions](https://docs.sourcegraph.com/dev/background-information/sourcegraph_extensions)).
```
Possibly this will be a section in the extensions registry soon-ish!